internal/cmd/schema_test.go

```diff
@@ -128,65 +128,69 @@ caveat test/some_caveat(someCondition int) {
 
 func TestSchemaCompile(t *testing.T) {
```

Comment:
I refactored these to be three separate tests because 1. you can't really use `require.ErrorAs` with a tabular test, at least in its current form (which may change in golang 1.26 but we're not there yet) and 2. one of the error assertions needed to be `ErrorIs`, not `ErrorAs`. There's a bit of duplicated code but I'm fine with that.